tasks/skylab/LoomUtils.wdl

```diff
@@ -297,98 +298,7 @@ task SingleNucleusOptimusLoomOutput {
 }
 
 
-task SingleNucleusSmartSeq2LoomOutput {
```

Comment:
@nikellepetrillo Should we leave the original task in place? We left the other Loom tasks in.

Reply:
@ekiernan I think we can leave it so it is consistent with other pipelines. Maybe one day we can clean up all unused tasks.